src/App.tsx

```diff
@@ -12,161 +16,20 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
```

Comment:
In point 5 of the task, it says:
Add the Navbar with 2 links Home and People. Active one should have the has-background-grey-lighter class;

I think that implements that we have to create a new component NavBar, however, I'm not 100% sure. Still, it would be good to have this component separately.